packages/icon-button/src/FzIconButton.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <button
+    type="button"
+    class="fz__iconbtn relative"
```

Comment:
non mi sembra che `fz__iconbtn` sia utilizzata

Reply:
corretto, ho lasciato le classi come referenza dello stile BEM che adotterei in caso di necessità, per rinforzare la coerenza e settare uno standard diciamo

Reply:
se adottiamo tailwind per definizione non abbiamo bisogno di uno standard come BEM

Reply:
Non sono d'accordo, possono tranquillamente convivere: sono due attrezzi e se in un caso usare Tailwind dovesse risultare particolarmente inconveniente, avere il secondo aiuta. Anche io preferisco l'approccio alla composizione di utility class di Tailwind, ma non illudiamoci che risolva ogni problema.

Chiaramente in questo caso la classe non serve se non per dare una sorta di "label" al pezzo di DOM e ricordare che in caso dovesse diventare troppo "ciccione", possiamo trasformarlo in un componente